common/exchange/rates.go

```diff
@@ -33,6 +33,9 @@ func ConvertCurrencyToCelo(exchangeRates common.ExchangeRates, currencyAmount *b
 	if feeCurrency == nil {
 		return currencyAmount, nil
 	}
+	if currencyAmount == nil {
```

Comment:
I would have hoped that we catch this earlier, how did you trigger this?

Reply:
I think I only triggered it while my tracing implementation was buggy.